qf/types/requests.proto

```diff
@@ -0,0 +1,172 @@
+syntax = "proto3";
+package types;
+option go_package   = "github.com/quickfeed/quickfeed/qf/types";
+option swift_prefix = "";
+
+import "qf/types/types.proto";
+
+// Message types that are used as requests and responses in API calls.
+
+message SubmissionLink {
+    types.Assignment assignment = 1;
+    types.Submission submission = 2;
+}
+
+message EnrollmentLink {
```

Comment:
This should be in the `types.proto` file (it is only used by the database ; not used in `qf.proto`).